dev/com.ibm.ws.microprofile.faulttolerance.cdi.2.0/.gitignore

```diff
@@ -0,0 +1,2 @@
+/bin/
+/generated/
```

Comment:
I don't think this .gitignore file is required ... the top level one covers bin and generated

Reply:
It's better than that. There's one at the top level and another in `dev`...

Either way, not needed here so removed.